recipes/dialign2/post-link.sh

```diff
@@ -1,2 +0,0 @@
-echo "In order to use dialign2 you have make sure that the environment variable DIALIGN2_DIR is set to $PREFIX/share/dialign2; will do this not via `export DIALIGN2_DIR=$PREFIX/share/dialign2`"
```

Comment:
This is not needed anymore?

Reply:
this is taken care of with the patch and during the build, no need for pre and post scripts anymore

Reply:
thx!